epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyModel.java

```diff
@@ -137,7 +183,54 @@ public void bind(T view, EpoxyModel<?> previouslyBoundModel) {
    *
    * @see EpoxyAdapter#onViewRecycled(EpoxyViewHolder)
    */
-  public void unbind(T view) {
+  public void unbind(@NonNull T view) {
+  }
+
+  /**
+   * Called when the model has changed but still represents the same item. Return true if you want
+   * to animate these changes manually in the viewholder instead of simple cross fading or whatever
+   * animation the {@link android.support.v7.widget.RecyclerView.ItemAnimator) does by default.
+   *
+   * @return true if you wish to animate changes in the model
+   */
+  public boolean canReuseViewHolder(@NonNull EpoxyModel<?> previouslyBoundModel) {
```

Comment:
Indeed, I just forgot to refactor it yesterday. But it's not necessarily the final name